images/stolon/Dockerfile

```diff
@@ -1,42 +1,65 @@
-FROM quay.io/gravitational/debian-venti:go1.11.5-stretch as builder
-ENV DEBIAN_FRONTEND=noninteractive \
-    TERM=xterm \
-    REPO_PATH=/gopath/src/github.com/gravitational/stolon
-WORKDIR $REPO_PATH
-COPY stolon .
+ARG PGVERSION=9.6
+# base build image
+FROM golang:1.13-buster AS build_base
```

Comment:
I've seen `golang:1.11` used in the primary Makefile - why not settle on a single version?

Reply:
All this PR is a bit WIP. So, yes, I agree, I am going to replace versions in Makefile and reuse it here as an argument to `docker build`.